src/components/NcListItem/NcListItem.vue

```diff
@@ -87,14 +87,16 @@
 			:name="'This is an active element with normal counter'"
 			:bold="false"
 			:active="true"
-			:details="'1h'"
 			:counter-number="44">
 			<template #icon>
 				<NcAvatar :size="44" user="janedoe" display-name="Jane Doe" />
 			</template>
 			<template #subname>
 				In this slot you can put both text and other components such as icons
 			</template>
+			<template #details>
+				<LinkIcon :size="16"/>
```

Comment:
should it not be 20 as that's our default?

Reply:
Same as bubble icon below, and close to the font size. But that's up to end developer anyway, I believe